vpr/src/place/move_utils.cpp

```diff
@@ -707,6 +726,13 @@ bool find_to_loc_centroid(t_logical_block_type_ptr blk_type,
     int cy_to = OPEN;
     bool legal = false;
 
+    if (is_cluster_constrained(b_from)) {
```

Comment:
might want to move the is_cluster_constrained inside the intersect routine -> run this way on one big circuit, and see if it makes any difference in cpu time

Reply:
moving this inside the intersect routine makes the place time go up, so i kept it as is